analyticsdata/src/run_report_with_aggregations.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Copyright 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+* Google Analytics Data API sample application demonstrating the usage of
+* metric aggregations in a report.
+* See https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport#body.request_body.FIELDS.metric_aggregations
+* for more information.
+
+* Usage:
+*   composer update
+*   php run_report_with_aggregations.php YOUR-GA4-PROPERTY-ID
+*/
+
+namespace Google\Cloud\Samples\Analytics\Data;
+
+// [START analyticsdata_run_report_with_aggregations]
+use Google\Analytics\Data\V1beta\BetaAnalyticsDataClient;
+use Google\Analytics\Data\V1beta\DateRange;
+use Google\Analytics\Data\V1beta\Dimension;
+use Google\Analytics\Data\V1beta\Metric;
+use Google\Analytics\Data\V1beta\MetricType;
+use Google\Analytics\Data\V1beta\MetricAggregation;
+
+function run_report_with_aggregations(string $propertyId)
+{
+    // [START analyticsdata_initialize]
+    // mports the Google Analytics Data API client library.
+    $client = new BetaAnalyticsDataClient();
+    // [END analyticsdata_initialize]
+
+    // Make an API call.
+    $response = $client->runReport([
+        'property' => 'properties/' . $propertyId,
+        'dimensions' => [new Dimension(
+            [
+                'name' => 'country',
+            ]
+        )],
+        'metrics' => [new Metric(
+            [
+                'name' => 'sessions',
+            ]
+        )],
+        'dateRanges' => [
+            new DateRange([
+                'start_date' => '365daysAgo',
+                'end_date' => 'today',
+            ]),
+        ],
+        'metricAggregations' => [
+    MetricAggregation::TOTAL,
+    MetricAggregation::MAXIMUM,
+    MetricAggregation::MINIMUM
+        ]
+    ]);
+
+    printRunReportResponseWithAggregations($response);
+}
+
+// Print results of a runReport call.
+function printRunReportResponseWithAggregations($response)
```

Comment:
Needs PHPDoc and typehint
```suggestion
/**
 * @param RunReportResponse $response
 */
function printRunReportResponseWithAggregations(RunReportResponse $response)
```

Reply:
This also will require an import at the top for `RunReportResponse`